libparsec/crates/protocol/schema/authenticated_cmds/shamir_create_shared_recovery_device.json5

```
@@ -0,0 +1,71 @@
+[
+    {
+        "major_versions": [
+            4
+        ],
+        "req": {
+            "cmd": "shamir_recovery_setup",
+            "fields": [
+                {
+                    "name": "setup",
+                    // Set to `None` to clear previous Shamir recovery setup
+                    "type": "RequiredOption<ShamirRecoverySetup>"
+                }
+            ]
+        },
+        "reps": [
+            {
+                "status": "ok"
+            },
+            {
+                // Certificate is not signed by the authenticated user, or timestamp is invalid
+                "status": "invalid_certification"
+            },
+            {
+                // Cannot deserialize data into the expected certificate, or inconsistency
+                // between  certificates and/or threshold
+                "status": "invalid_data"
+            },
+            {
+                // Future evolution 1: Shamir recovery has already been setup, should ask your admin to reset it first !
+                "status": "already_set"
```

Create commands (`block_create`, `device_create`, `realm_create`, `user_create`) have a status named `{entity}_already_exists` (i.e. `user_alread_exists`). Maybe we should use the same logic here. 
That said, I'm not sure what the entity should be:
```suggestion
                "status": "shamir_already_exists"
```
```suggestion
                "status": "shamir_setup_already_exists"
```

It can also be just `already_exists` although it may become ambiguous if we add more arguments to the request.